modules/nf-cid/src/main/nextflow/data/config/DataStoreOpts.groovy

```diff
@@ -30,14 +26,10 @@ import groovy.transform.CompileStatic
 @CompileStatic
 class DataStoreOpts {
 
-    final Path location
-    final Path logLocation
+    final String location
```

Comment:
Store location not necessarily needs to be a file system path. Changed to a generic `String`